docs/features/duplicate_granule_handling.md

```diff
@@ -0,0 +1,67 @@
+---
+id: duplicate_granule_handling
+title: Cumulus Duplicate Granule Handling
+hide_title: false
+---
+
+Collections and Workflows can be configured to assign a unique `granuleId` to each Granule. This is useful in cases where two Granules share the same `granuleId` but must both be ingested into the Cumulus system. Previously, a Granule with a conflict on the `granuleId` key would cause an ingest failure for that Granule.
+
+## Granule Uniquification
+
+Cumulus has implemented a set of changes that allow multiple Granules with identical `granuleIds` to be ingested and managed independently. This is done by replacing the `granuleId` with a unique value and storing the original `granuleId` value as a new field, `producerGranuleId`. This new `producerGranuleId` field can then be used to track and correlate the Granule to the original data holdings.
+
+### Granule Uniquification Algorithm and Task Component
+
+The process Cumulus Core uses to generate a unique `granuleId` can be [found here](https://github.com/nasa/cumulus/blob/feature/duplicate-granule-924/packages/ingest/src/granule.ts#L479-L487).
+
+To make the process easier, Cumulus Core also provides a Task Component that will uniquely identify each Granule in a payload and return a modified Granule object containing a unique `granuleId` and a `producerGranuleId` containing the original `granuleId`.
+
+## Configuration
+
+### Collection Configuration
+
+Cumulus Collections can be configured to replace the Granule's existing `granuleId` with a unique value. The originnal `granuleId` will then be stored in a new field, `producerGranuleId`.
+
+To configure a Collection to support uniquely identifying Granules, the following params are used in the Collection's JSON:
```

Comment:
But the uniquification can be configured in the other places like rule.meta, or hardcode the value in workflow task configuraton.